app/src/main/java/blockeq/com/stellarwallet/encryption/KeyStoreWrapper.kt

```diff
@@ -64,6 +64,16 @@ class KeyStoreWrapper(private val context: Context) {
      */
     fun removeAndroidKeyStoreKey(alias: String) = keyStore.deleteEntry(alias)
 
+    /**
+     * Clears all aliases in the Android Key Store
+     */
+    fun clear() {
```

Comment:
could you write a small unit test for this?

Reply:
Yup!